### ISSUE TITLE
bug: Search bar in toolbar not centered iOS

### ISSUE THREAD

**Issue by LiveXenon** [ionic-team/ionic#19495]
# Bug Report

**Ionic version:**
5.4.1

**Current behavior:**
After updating the packages to the latest version I noticed that the search bar is no longer centered vertically.

Previous version of the package:
```json
    "@ionic-native/native-storage": "^5.7.0",
    "@ionic-native/photo-viewer": "^5.12.0",
    "@ionic-native/splash-screen": "^5.0.0",
    "@ionic-native/status-bar": "^5.0.0",
    "@ionic-native/twitter-connect": "^5.7.0",
    "@ionic/angular": "^4.1.0",
```

<img width="377" alt="Schermata 2019-09-29 alle 20 09 24" src="https://user-images.githubusercontent.com/34724827/65837224-1ba25880-e2f5-11e9-8657-0b494474f308.png">
<img width="448" alt="Schermata 2019-09-29 alle 20 08 25" src="https://user-images.githubusercontent.com/34724827/65837225-20ffa300-e2f5-11e9-86f2-dbf44dff5800.png">

**Related code:**
`<ion-header>
  <ion-toolbar color="primary">
    <ion-buttons slot="start">
      <ion-back-button text=""></ion-back-button>
    </ion-buttons>
    <ion-searchbar placeholder="{{ 'searchUser' | translate }}" (ionChange)="search($event)" [debounce]="500">
    </ion-searchbar>
  </ion-toolbar>
</ion-header>`

**Ionic info:**
Ionic:

   Ionic CLI                     : 5.4.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.10.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Capacitor:

   Capacitor CLI   : 1.0.0
   @capacitor/core : 1.0.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.1.0, ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 20 other plugins)

Utility:

   cordova-res : 0.3.0 (update available: 0.6.0)
   native-run  : 0.2.8 

System:

   ios-sim : 8.0.2
   NodeJS  : v10.15.0 (/usr/local/bin/node)
   npm     : 6.11.2
   OS      : macOS Mojave
   Xcode   : Xcode 10.3 Build version 10G8

**Comment by rossholdway** [comment]
@LiveXenon Should be fixed in #19502

**Comment by brandyscarney** [comment]
Thanks for the issue! The searchbar was moved to the top of the toolbar to match the iOS spec, see my comment here: https://github.com/ionic-team/ionic/issues/19502#issuecomment-537522707

I'm adding a fix for the back button now.
